icka.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+#--------------#
+# Instructions #
+#--------------#
+# Edit the script replacing any occurrence of `$user` with your shell username,
+# your email, password and correct paths to the binary
+##
+# Make this file executable with:
+#   chmod +x /home/$user/icka.sh
+##
+# Add the following to your crontab, to make it run every hour, so you never disconnect:
+#   0 * * * * /home/$user/icka.sh >/dev/null 2>&1
+##
+# Enjoy!
+#---------------------#
+# End of instructions #
+#---------------------#
+
+
+##---------------##
+## Configuration ##
+##---------------##
+
+#------------------------#
+# IRCCloud account email #
+#------------------------#
+ickaEmail="email@example.com"
+
+#---------------------------#
+# IRCCloud account password #
+#---------------------------#
+ickaPassword="foobar"
+
+#-------------------------#
+# Path to the icka binary #
+#-------------------------#
+ickaBinary="/home/$user/go/bin/icka"
```

Comment:
Should this be changed to `$HOME/go/bin/icka`?

Reply:
To be honest, I'm unsure.

I assume that we can customize the install location and I don't like assuming normal installations and I'm a fan of complete paths.

That's why I suggest on top of the script to replace `$user` with the shell username.

Reply:
I just edited the path to the log so it looks consistent across the script.

I still prefer that the user edit the path to the log file and binary instead assuming things, since behaviour can be different on Arch, etc.

Reply:
Looks good!

I squashed your commits and merged it, will close this PR now, thanks!